Build home page and toolbar & Add styles

s2jack opened the pull request:
Add linters
Create Home Page
Create Toolbar
Create Status Bar
Add greetings message
Add header
Add social media icons
Create images folder
Create fonts folder
Add fonts
Add images
Fix linter errors

s2jack committed:
Create home page

Create gitignore file
Create index.html
Create styles.css
Add README
Add gitignore
Add linters
Create fonts folder
Add fonts

s2jack committed:
Create toolbar and home page & add styles

Create homepage
Create toolbar
Create statusbar
Add greetings message
Add social media icons
Add background images
Add Roboto font

s2jack committed:
Fix stylelint errors

Fix stylelint errors

s2jack committed:
delete junk files fix linters

delete hint report junk file
fix linter errors

usorfaitheloho commented on index.html:
```diff
@@ -0,0 +1,50 @@
+    <title>Selcuk Guler's Personal Portfolio website</title>
+  </head>
+  <body>
+    <main>
+      <div id="topbar">
+        <div id="appbar">
+          <div id="timer-cont">
+            <div id="timer-text">9:41</div>
+          </div>
+          <div id="statusbar">
+          </div>
+        </div>
+        <div class="container" id="toolbar">
+          <div id="logo-placeholder"></div>
+          <div id="hamburger-cont">
+            <div id="hamburger-icon"></div>
+          </div>
+        </div>
+      </div>
```
Please implement these changes for the header section of your page 👍 

- [x] Kindly use a `header` tag for the header of your website which depicts introductory content majorly for good HTML semantics and a clean code 😄 , and usually the `header` tag comes before the `main` tag. You can check this [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more

- [x] Please use a `nav` tag for your hamburger menu ie, line `22 - 24` should be inside a `nav` tag 😄

usorfaitheloho commented on index.html:
```diff
@@ -0,0 +1,50 @@
+          </div>
+        </div>
+      </div>
+      <div id="home">
+        <div id="greeting-container">
+          <h1 class="greetings">
+            <span class="black-text">Hey there.<br>I'm Selçuk.</span>
+            <br>
+            <span class="green-text">I'm a software<br>developer</span>
+          </h1>
+        </div>
+        <div id="message-cont">
+          <span id="support-txt"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.</span>
+        </div>
+        <div id="sm-icons-cont">
+          <span class="sm-icons" id="twitter"></span>
+          <span class="sm-icons" id="linkedin"></span>
+          <span class="sm-icons" id="medium"></span>
+          <span class="sm-icons" id="github"></span>
+          <span class="sm-icons" id="angellist"></span>
+        </div>
+        <div id="home-bg-img"></div>
+      </div>
```
Please implement these changes for this section of your project :

- [x] kindly use a `section` tag for the container of this section of your code, as it represents a generic standalone section of the document , you can check this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info 😄
- [x] kindly use a `p` tag for this paragraph on `line 35-38` for semantic reasons
- [x] please the list of icons should be listed like this ⬇️ 
   > 
              <ul>
                 <li> <a href="#"> <span class="sm-icons" id="twitter"></span>  </a> </li> 
                   .
                   .
                   .
             </ul>

usorfaitheloho commented on styles.css:
```diff
@@ -0,0 +1,186 @@
+  background-image: url("images/sm-icons/angellist.png");
+  width: 14.17px;
+  height: 19.62px;
+}
```
Kindly implement the following design changes :

- [x] Please the introductory text font color is not in the right color of `#172B4D`, kindly change it from the black color to the one indicated in the design which I listed
- [x]  please also change the font colour of the paragraph `I can help you build a product feature...` to the color i mentioned above which is the same color in the Figma template 😄 

Your design :
![image](https://github.com/s2jack/create-portfolio-website1/assets/20537712/ba78d454-4bec-4965-b127-3887223e3b59)

Figma design :
![image](https://github.com/s2jack/create-portfolio-website1/assets/20537712/056fd945-c00e-4176-810a-2cfeb56ed528)

usorfaitheloho reviewed (changes requested):
> Hi @s2jack  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

 ![](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966) [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Good UI design ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

usorfaitheloho reviewed (changes requested):
> Hi @s2jack  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

 ![](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966) [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Good UI design ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

s2jack committed:
Fix review requests

Add header tag
Add nav tag
Add home section tag
Add p tag
Add ul and li elements for social media icons
Change colors of dark texts

taldr27 commented on index.html:
```diff
@@ -0,0 +1,50 @@
+          <div id="statusbar">
+          </div>
+        </div>
+        <div class="container" id="toolbar">
+          <div id="logo-placeholder"></div>
+          <nav id="hamburger-cont">
+            <div id="hamburger-icon"></div>
+          </nav>
+        </div>
```
- [ ] Fantastic work on implementing the requested changes from the previous reviewer! However, there is **one remaining change that has not been addressed yet**. Could you please consider using a `header` tag instead of a `div`? Utilizing a header tag aligns with **HTML best practices**. You can find more information about it in this [link](https://www.w3schools.com/tags/tag_header.asp).

  ![image](https://github.com/s2jack/create-portfolio-website1/assets/107952724/e454a3b7-059c-4cbd-a52d-ccf3a3ea706d)

taldr27 reviewed (changes requested):
# Status: Required Changes ♻️♻️

Hi @s2jack, 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but some are still not addressed yet.

![almost](https://media.giphy.com/media/t8YPf3ACKeouLzEIvr/giphy.gif)

## Highlights 🎉 
✅ Professional Readme. 
✅ Linters tests passing.
✅ Correct use of GitHub Flow.
✅ Great work with your design.
✅ Incorporates feedback from previous code reviews.

 ## Required Changes ♻️
_Check the comments under the review._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in @taldr27 your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). <img align="right" src="https://media.giphy.com/media/kReKcfrs1YoTmt2AQt/giphy.gif" alt="gif"  width="3%"/>_

s2jack committed:
Fix code review requests

Cancel div tag that contains header tag
Use only header tag

Qoosim commented on index.html:
```diff
@@ -0,0 +1,48 @@
+  </head>
+  <body>
+    <main>
+      <div id="topbar">
+        <div id="appbar">
+          <div id="timer-cont">
+            <div id="timer-text">9:41</div>
+          </div>
+          <div id="statusbar">
+          </div>
+        </div>
+        <div class="container" id="toolbar">
+          <div id="logo-placeholder"></div>
+          <nav id="hamburger-cont">
+            <div id="hamburger-icon"></div>
+          </nav>
+        </div>
```
- [x] This is the appropriate place for the header tag. The haeder tag houses the logo and the hamburger of your page on the mobile. The header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation(in this case, your hamburger). Kindly change the generic `div` with the `header` tag

###  Items inside your header
![your-toolbar](https://github.com/s2jack/create-portfolio-website1/assets/26597432/95790ec4-7761-4659-b53b-07e2435bd4e2)

### Items to be inside the header tag
![project-rqt](https://github.com/s2jack/create-portfolio-website1/assets/26597432/4019be80-30bb-4388-8716-3f07c1995104)

- [x] As mentioned by the previous reviewer, it is not a good practice for the  `header` tag to be inside the main tag. Kindly address that too.

Qoosim reviewed (changes requested):
#  *Status: Required Changes* :recycle: 

Hi @s2jack, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 

![](https://camo.githubusercontent.com/38ce8c8b213b08e2d9bb85a0b79702ad512f3910f803ec3d7917b4e0110c9044/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966)

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

s2jack committed:
Add header tag

Add header tag

s2jack committed:
Get out header element from main element

taldr27 reviewed (approved):
# Status: Approved! 🟢 
Hi @s2jack, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

![congrats](https://media.giphy.com/media/3o6fIUZTTDl0IDjbZS/giphy.gif)

## Highlights 🎉
✅ Professional and customized README. 🤓
✅ Linters tests passing. 👍
✅ Correct use of GitHub Flow. 🙌
✅ Great design. 💯
✅ Incorporates feedback from previous code reviews.🎯

-----
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). <img align="right" src="https://media.giphy.com/media/kReKcfrs1YoTmt2AQt/giphy.gif" alt="gif"  width="3%"/>_
